templates/nginx-conf.j2

```diff
@@ -47,6 +47,10 @@ http {
     sendfile        on;
     #tcp_nopush     on;
 
+    {% for http_conf_line in nginx_proxy_conf_http -%}
```

Comment:
It's a Jinja2 way of saying 'strip all whitespace' and vastly improves the resulting file:

see before, after:

![screen shot 2017-07-31 at 13 52 49](https://user-images.githubusercontent.com/2727871/28778530-9321fd1a-75f7-11e7-870c-274dd8f1c458.png)
![screen_shot_2017-07-31_at_11_34_40](https://user-images.githubusercontent.com/2727871/28778529-9320e61e-75f7-11e7-87fa-ab5722268e6d.png)